ccp/notifications/notify.py

```diff
@@ -111,18 +136,18 @@ def notify(self,
         # possible values are ["success", "failed"]
         status = True if status == "success" else False
 
-        # get the repository name (without tag)
-        repository = "https://" + build_info.get("REGISTRY_URL") + \
-            "/" + image_name.split(":")[0]
-
         cause = build_info.get("CAUSE_OF_BUILD")
 
         # subject should have image_name without registry
         subject = self.subject_of_email(
-            status, image_name, build_info.get("REGISTRY_URL"))
+            status, image_name, build_info.get("REGISTRY_URL"),
+            build_info.get("REGISTRY_ALIAS"))
 
         # body should have repository name (without tag)
-        body = self.body_of_email(status, repository, cause)
+        body = self.body_of_email(
+            status, image_name, cause,
+            build_info.get("REGISTRY_URL"),
+            build_info.get("REGISTRY_ALIAS"))
 
```

Comment:
:+1:

Reply:
:+1:

Reply:
The way we are fetching the parameters from build_info in notifications module,
at some point later, we'll be fetching parameters in same way in index_reader.py and at that time, the build will be of the seed-job.
Its just that, index_reader implementation happened first and we implemented fetching the build_info while working on notification piece.

One of the concern I have for index_reader.py is the growing number of arguments the script is invoked with. In this PR, one more parameter is being added.